JapaneseFilter/sections/css_extended.txt

```diff
@@ -95,6 +95,7 @@ otonasalone.jp,sumaiweb.jp#?#._popIn_infinite_page > a._popIn_recommend_article_
 blog.fc2.com#?#div[class$="_outline"] > div > div.plugin-freearea > a[href^="https://www.amazon.co.jp/dp/"][target="_blank"]:not([name]):upward(3)
 blog.fc2.com#?#div[class$="_outline"] > div > div.plugin-freearea > ins.adsbygoogle:upward(3)
 jobcatalog.yahoo.co.jp#?#div[class*="__"] > div[id$="ad1"]:upward(1)
+pokemonbbs.com#$?#div[style="height:100px"] > script[src*="https://pagead2"]:upward(1) { remove: true; }
```

Comment:
Is it really needed? What about `pokemonbbs.com##div[style="height:100px"]`? It works fine on my side.

Reply:
Sorry,I mistake that.
`pokemonbbs.com##div[style="height:100px"]` works fine.